content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -14,70 +14,116 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 The following example demonstrates automatic and manual instrumentation via an
 ASP.NET Core app.
 
-First, install all the required packages:
+First, create your basic ASP.NET Core site:
+
+```shell
+dotnet new mvc
+```
+
+Next, Add the Core OpenTelemetry packages
 
 ```shell
 dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
-dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
-dotnet add package OpenTelemetry.Instrumentation.Http --prerelease
-dotnet add package OpenTelemetry.Instrumentation.SqlClient --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-Note that the `--prerelease` flag is required for all instrumentation packages
-because they are all are pre-release.
+Now let's add the automatic instrumentation packages for ASP.NET Core. This will
+give us some automatic spans for each HTTP request to our app.
+
+```shell
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
+```
 
-This will also install the `OpenTelemetry` package.
+_Note that as the Semantic Conventions for attribute names are not currently
+stable the instrumentation package is current not in a released state. That
+doesn't mean that the functionality itself is not stable. This means that you
+need to use the `--prerelease` flag, or install a specific version of the
+package_
 
-### Tracing
+This will also install the `OpenTelemetry` core packages.
```

Comment:
potentially misleading, as lines above explicitly installs "Core" packages (console,ext.hosting).. And installing aspnetcore instrumentaion wont install them.